data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -1964,12 +1964,10 @@ _ "After using this attack, you can’t use it during your next two turns."#endd
         value = 2
         id = natures_aid
         affect_allies=yes
-        name = _ "nature's aid +2"
-        female_name = _ "female^nature's aid +2"
-        description = _ "Allows the nature to heal units standing in forest or fungus at the beginning of our turn.
-
-A unit cared for by nature itself may heal up to 2 HP per turn, or stop poison from taking effect for that turn.
-A poisoned unit cannot be cured of its poison by a healer, and must seek the care of a village or a unit that can cure."
+        name = _ "nature’s aid +2"
+        female_name = _ "female^nature’s aid +2"
+        description = _ "All units on this unit’s side, regardless of distance, will heal themselves for 2 HP when starting their turn in forest of fungus." + "
```

Comment:
'forest **or** fungus'? Same for line 1996.

Looks like there's only one line break instead of two between the translated texts. So I think this will appear as:
```
All units on this unit’s side, regardless of distance, will heal themselves for 2 HP when starting their turn in forest of fungus.
This ability will not cure an affected unit of poison, however, only delay its effect.
```

Instead of:
```
All units on this unit’s side, regardless of distance, will heal themselves for 2 HP when starting their turn in forest of fungus.

This ability will not cure an affected unit of poison, however, only delay its effect.
```

I thought the latter is what you intended. This applies to lines 1996 and 2020 as well.

Reply:
You are a godsend.